django/utils/datastructures.py

```diff
@@ -131,18 +131,21 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
-        """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
-        """
+    def _getlist(self, key, default=None):
```

Comment:
Maybe a comment or docstring could help explain why the underscore version exists?

Also, I wonder if the fix should apply at the `MultiValueDict` level and not just to `QueryDict`?